rotkehlchen/db/history_events.py

```diff
@@ -332,12 +332,12 @@ def get_history_events_count(self, cursor: 'DBCursor', query_filter: HistoryEven
     def get_value_stats(
             self,
             cursor: 'DBCursor',
-            query_filter: HistoryEventFilterQuery,
-    ) -> tuple[FVal, list[tuple[Asset, FVal, FVal]]]:
+            query_filters: str,
+            bindings: list[Any],
+    ) -> tuple[FVal, list[tuple[str, FVal, FVal]]]:
```

Comment:
I edited this because it is a function that we also use for the kraken stats here https://github.com/rotki/rotki/blob/40577eba978a6c988157b4637e780d89caa90a6e/rotkehlchen/api/rest.py#L3645

The reason to pass the query and binding instead of the filter is because the liquity query is complicated and creating a filter for it would just complicate things. With the refactor of the history events this filter would be much easier to do and I can add a TODO to simplify this then

Reply:
I also disliked this and this is why I proposed the refactor to the history events. I prefer to make keep this change like this for now and revert it later (is really easy) once the filters can be properly used for this because it unblocks the frontend to work in this issue

Reply:
All right. Fair enough.